### DISCUSSION TITLE
feature: Tweet embed cards

### DISCUSSION THREAD

**Discussion by orimdominic** [adityatelange/hugo-PaperMod#882] [Q&A] [answered]
Hello!

I'd like to know if we have the feature to display/embed a tweet on the blog. I attached an image of what I mean.
Do we already have this? Are there any plans to include this?

#### What I tried
I tried using the code from the Twitter embed site on the markdown for the blogpost but nothing was displayed.

![tweet_embed](https://user-images.githubusercontent.com/37955249/163609010-d0c2d02f-1283-428e-b8d9-72a85e400545.png)

**Comment by adityatelange** [comment, marked as answer]
https://gohugo.io/content-management/shortcodes/#tweet

**Reply by orimdominic** [reply]
Ha! Sweet! :raised_hands:
